atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -110,6 +111,7 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
     """
 
     EXCLUDE_FILENAME = 'exclude-platform'
+    UNREACHABLE_CLUSTER_LOAD = -1
```

Comment:
Sure, but in either case it creates a special case that must be known to the caller. Having `-1` is no better than `object()` in that sense.